articles/qdk-relnotes.md

```diff
@@ -19,8 +19,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
-## Version 0.15.2103.133969
 
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+- Added support for more types for 'w' and 'indices' inputs for QIO Python SDK.
```

Comment:
I would rephrase this as:
"Added support for NumPy types in coefficient definitions for problems in QIO Python SDK."